### ISSUE TITLE
Trunk API appears to be broken for recently pushed podspecs

### ISSUE THREAD

**Issue by marmelroy** [CocoaPods/CocoaPods#6184]
* [x] I've read and understood the [*CONTRIBUTING guidelines and have done my best effort to follow](https://github.com/CocoaPods/CocoaPods/blob/master/CONTRIBUTING.md).

# Report

I tried to load a shields.io Cocoapods shield on GitHub and received an "invalid pod" error. This is true for any podspec pushed in the last few days. I believe it's related to the recent sharding.

You can see the "invalid" pod shield on many GitHub OSS projects including https://github.com/Alamofire/Alamofire

Shields.io is using the following URL template to load data about the pod and it appears that this API is broken for recently updated podspecs:
```
apiUrl = 'https://trunk.cocoapods.org/api/v1/pods/' + spec + '/specs/latest';
```

**Comment by DanToml** [comment]
@marmelroy Good catch, thanks!

**Comment by marmelroy** [comment]
Thanks @DanToml. Let me know if there's anything I can do to help.

**Comment by GeroHerkenrath** [comment]
Just as a pointer for those doing a google search: This has already been asked on [Stack Overflow](http://stackoverflow.com/q/40599432/710041) as well. I'm going to post a link back to this issue here as well.

Btw, the asking user there says that Alamofire worked yesterday, which I can confirm. So it seems (due to caching/scheduled updates on the shields?) it creeps over the pods slowly (and publishing new versions triggers the "loss" of the badge instantly).
For me it's not a biggy, though, especially when it's something global that you will eventually fix. :)

**Comment by codytwinton** [comment]
+1

**Comment by segiddins** [comment]
This should probably be filed on the shields.io repo, as it has nothing to do with the CocoaPods command line tool -- shields will need to update the URL they use.

**Comment by marmelroy** [comment]
Thanks @segiddins. Is there a new URL for the trunk API then? 

I'm happy to fix it with a PR on the shields.io repo but I can't seem to find any documentation for a new URL template. 

They are currently using :
```apiUrl = 'https://trunk.cocoapods.org/api/v1/pods/' + spec + '/specs/latest';```

**Comment by codytwinton** [comment]
@marmelroy - for what it's worth, I think `/latest` was removed as a valid option or changed.

This works fine: https://trunk.cocoapods.org/api/v1/pods/Alamofire/specs/4.1.0

Perhaps something recently changed on the Cocoapods Trunk that removed `latest` from mapping to the most recent pod.

**Comment by codytwinton** [comment]
Also, I believe I may have found the file where the issue may be located: https://github.com/CocoaPods/trunk.cocoapods.org/issues/203

**Comment by loudmouth** [comment]
I'm also experiencing the same issue on my repo ([repo on Cocoapods.org](https://cocoapods.org/pods/ContentfulDeliveryAPI), and repo on [Cocoadocs.org](http://cocoadocs.org/docsets/ContentfulDeliveryAPI/)), but I have not been able to find a solution on all the linked github issues or on the shields.io repo.

Going off of Cody's comment above, I noticed that the following link resolves properly:
https://trunk.cocoapods.org/api/v1/pods/ContentfulDeliveryAPI/specs/2.0.0, while the path ending in `latest` results in a 404.

I pushed my pod with Cocoapods 1.2.0 on February 22nd. Is it possible that there was a regression introduced somewhere despite the linked above fix https://github.com/CocoaPods/trunk.cocoapods.org/issues/203 ?
